src/app/modules/core/services/beacon-node.service.ts

```diff
@@ -11,9 +11,20 @@ import {
   NodeConnectionResponse,
 } from 'src/app/proto/validator/accounts/v2/web_api';
 
+import {
+  ValidatorBalances,
+} from 'src/app/proto/eth/v1alpha1/beacon_chain';
+
 const POLLING_INTERVAL = 3000;
 const BEACON_API_SUFFIX = '/eth/v1alpha1';
 
+export class BeaconState {
+  beaconNodeEndpoint$: BehaviorSubject<string> = new BehaviorSubject(undefined);
```

Comment:
Is behavior subject what we want here? Feels weird to initialize it with undefined or false. Perhaps `ReplaySubject` works best here

Reply:
the advantage of a BehaviorSubject is that it keeps beeing accessable after it changed ... think this is needed if we only want to load the endpoint once